samples/stopwatch-coroutines/src/test/java/com/instacart/formula/stopwatch/StopwatchFormulaTest.kt

```diff
@@ -0,0 +1,33 @@
+package com.instacart.formula.stopwatch
+
+import com.google.common.truth.Truth.assertThat
+import com.instacart.formula.coroutines.toFlow
+import kotlinx.coroutines.flow.first
+import kotlinx.coroutines.flow.launchIn
+import kotlinx.coroutines.flow.onEach
+import kotlinx.coroutines.flow.withIndex
+import org.junit.Rule
+import org.junit.Test
+
+@ExperimentalStdlibApi
+class StopwatchFormulaTest : CoroutineTest {
+
+    @get:Rule
+    override val coroutineRule = CoroutineTestRule()
+
+    @Test
+    fun `increment 5 times`() = test {
+
+        StopwatchFormula(this)
```

Comment:
It would be good to have such a tests within `formula-coroutines` module to cover `toFlow()` and `FlowStream.fromFlow {}`.

Reply:
I added test for `FlowRuntime` and `FlowStream`. I know you were trying to decide if it still made sense to have per runtime specific tests, but we can always remove them. I added `TestFlow` to serve the purpose of Rx's `TestObserver`. I know square came out with [Turbine](https://github.com/cashapp/turbine), but it felt like an overkill.